Chandan/chandan_practice.c

```diff
@@ -1,106 +1,199 @@
+   printf("\n The division of C and B ");
+   divi = *r / *q;
+   printf("\n The quotient = %f \n", divi); 
+   
+   printf(" \n The even and Odd numbers of X, Y and Z :");
+   if( x%2 == 0)
+   printf ("\n X is an even number which is: %d ", x);
+   else
+   printf("\n X is an odd number which is : %d ", x);
+   
+   if( y%2 == 0)
+   printf ("\n Y is an even number which is: %d", y);
+   else
+   printf("\n Y is an odd number which is : %d", y);
+   
+   if( z%2 == 0)
+   printf ("\n Z is an even number which is: %d",z);
+   else
+   printf("\n Z is an odd number which is : %d \n", z);
+   
+   
+   printf(" \n The greatest of A, B and C");
+   if(a>b && a>c)
+   printf(" \n A is greater which is %f \n", a);
+   else if( b>a && b>c)
+   printf(" \n B is greater which is %f \n", b);
+   else
+   printf("\n C is greater which is %f \n", c);
+   
+   printf(" \n The greatest of X, Y and Z");
+   if(x>y && x>z)
+   printf(" \n X is greater which is %d \n", x);
+   else if( y>x && y>z)
+   printf(" \n Y is greater which is %d \n", y);
+   else
+   printf("\n Z is greater which is %d \n", z);
+  
+   printf("\n The area of triangle A, B and C");
+   u = (sum)/ 2; 
+   //area = sqrt(u * (u - a) * (u - b) * (u - c));
+   area=10.0;
+   printf("\n The area = %f \n", area);
+   
+   printf("\n The area of circle C ");
+   cir = 2 * 3.14 * c;
+   printf("\n Area = %f \n", cir);
+   
+   printf("\n The area of circle B ");
+   cirb = 2 * 3.14 * b;
+   printf("\n Area = %f \n", cirb);
+   
+   printf("\n The area of circle A ");
+   cira = 2 * 3.14 * a;
+   printf("\n Area = %f \n", cira);
+   
+   printf( " \n The area of rectangle length A and Breadth C");
+   rec = a * c;
+   printf("\n  Area = %f \n ", rec);
+   
+   printf(" \n To find the HCF and LCM of X,Y and Z");
+   
+   
+   for(i = 1; i <= x || i <= y || i< z ; i++) 
+   {
+       
+   if( x%i == 0 && y%i == 0 && z%i ==0 )
+      HCF = i;
+   }
 
-int getstarttime()
-{
-    int starttime;
-    starttime = 1;
-    return starttime;
-}
+   printf("\n HCF is = %d \n", HCF);
+   
+   
+   LCM = 0;
 
-int getendtime()
-{
-    int endtime;
-    endtime = 32767;
-    return endtime;
-}
+   if(x > y && x >z)
+      max = step = x;
+   else if (y> x && y> z )
+      max = step = y;
+   else 
+      max = step = z;
 
-int delay()
-{
-   int a = getstarttime();
-   int b = getendtime();
-   int c = a;
-   int d = b;
-   
-   for (c = 1; c <= 32767; c++)
-       for (d = 1; d <= 32767; d++)
-       {}
-       
-   return 0;
-}
+   while(1) 
+   {
+      if(max%x == 0 && max%y == 0 && max%z == 0) 
+      {
+         LCM = max;
+         break;    
+      }
 
+      max += step;
+   }
 
-void updateTimes()
-{
-        end = time(0);
+   printf(" LCM is=  %d \n", LCM);
+   
+   printf(" \n The program for palindrom");
+   
+    printf(" \n The string is: RADAR\n ");
+
+    for (j = 0;s[j] != '\0';j++)
+        {
+            g = s[j];
+            push(g);
+        }
+    for (j = 0;j < (strlen(s) / 2);j++)

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/manual/e-mock/CProgramming-Team6/pullRequest?prid=5597669)